scripts/js/interfaces.js

```diff
@@ -46,21 +47,32 @@ $(function () {
 
       const status = `<span class="${carrierColor}">${stateText}</span>`;
 
+      let master = null;
+      if (interface.master !== undefined) {
+        // Find interface.master in data.interfaces
+        master = data.interfaces.find(obj => obj.index === interface.master).name;
+      }
+
+      // Show an icon for indenting slave interfaces
+      const indentIcon = master === null ? "" : "<i class='fa fa-diagram-project fa-fw'></i> ";
+
       var obj = {
-        text: interface.name + " - " + status,
+        text: indentIcon + interface.name + " - " + status,
         class: gateways.has(interface.name) ? "text-bold" : null,
         icon: "fa fa-network-wired fa-fw",
```

Comment:
2. use the normal icon for master interfaces, but use an empty icon for child interfaces:
```suggestion
        icon: master === null ? "fa fa-network-wired fa-fw" : "fa fa-fw",
```